### ISSUE TITLE
support for drop files inside of Grid

### ISSUE THREAD

**Issue by elfwine** [vitmalina/w2ui#1355]
First of all, thank you for this easy to adapt UI framework.
It so happen that I need to support drop file inside of grid, where it will check whether the same filename is existing on grid records. If it exist, a popup edit/upload form will display, if not a popup add/upload form will display before the user decided whether to add as new or update existing record.

Any help is much appreciated.
By the way, I'm using version 1.4.3 as the latest version 1.5x is still a bit buggy on IE 11.

Thanks.

**Comment by spizzi77** [comment]
I don't understand well the problem because i don't understand if you want control records on the refresh or control records after a modal or popup opening, btw i try to answer you, to loop every records presents in the grid you will do in the refresh method, supponing that in columns field name a row "intestazione" on the refresh you do:

        onRefresh:function(event){
            event.onComplete= function(event){
                //eliminiamo il footer-left del tempo che ci mette a recuperare i dati ma lasciamo il footer right del numero righe
                $('.w2ui-footer-left').hide();
                records = this.records;

                    $(records).each(function(event){

                                       intestazione=this.intestazione,
                                         if (intestazione == 'YOURCONTROL'){
                                               alert('present in table');
                                          }
                                });
}}

**Comment by elfwine** [comment]
Thank you for your response. 
I solved it on my own, I had to implement a file upload that support drag and drop to grid's area. The grid's records has a field of filename, when a file drop to grid's area, I used the grid's function 'find', where it will look for record/s that has/have same filename of the file dropped. 
When there is/are recid/s retrieved, using the first recid, I will then retrieve that record information.
Then, pass it to the form in a popup whether the user has to update (if there's a record) or add new record information, submit it to the server including the file (uploaded). When no errors return, new data is fetch from the server and refresh the grid.

I'm still learning how this whole framework works, I'm glad to see this framework as it made my work easier. ^^ cheers! ;)
